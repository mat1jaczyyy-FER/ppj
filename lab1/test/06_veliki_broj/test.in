12346546756867898707851234123451089766644
