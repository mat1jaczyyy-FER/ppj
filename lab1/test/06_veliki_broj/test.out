BROJ 1 12346546756867898707851234123451089766644
